back/src/test/java/com/ubikloadpack/jmeter/ulp/observability/server/ULPObservabilityConfigServletTest.java

```diff
@@ -0,0 +1,51 @@
+package com.ubikloadpack.jmeter.ulp.observability.server;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import com.ubikloadpack.jmeter.ulp.observability.util.Util;
+
+public class ULPObservabilityConfigServletTest extends AbstractConfigTest {
+	@Test
+	@DisplayName("When a GET request is sent, expect ok and json as content type")
+	public void whenGetRequestExpectOkAndJsonContentType() throws Exception {	

```

Comment:
This test seems redundant with whenSendingGetRequestToConfigEndpointExpectConfigurationAsJsonFormat

Reply:
After a thought, yes, it can be removed. My goal was just to have a small test that verifies the status and the content type received from the server without considering the received configuration.